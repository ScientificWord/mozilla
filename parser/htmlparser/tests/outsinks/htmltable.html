<html>
<head>
<title>HTML To Text Test Page</title>
</head>

<body>
Below is a table.<br>

<table>
<tr><td>Row 1 Col 1<td>Row 1 Col 2</td><td>Row 1 Col 3</tr>
<tr><td>Row 2 Col 1 <td>Row 2 Col 2</td><td> Row 2 Col 3</tr>
<tr><td>Row 3 Col 1</td><td>Row 3 Col 2</td><td>Row 3 Col 3</tr>
</table>
Here is after table.
</body>
</html>
