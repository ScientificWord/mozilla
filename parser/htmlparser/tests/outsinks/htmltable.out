Below is a table.
Row 1 Col 1	Row 1 Col 2	Row 1 Col 3
Row 2 Col 1 	Row 2 Col 2	Row 2 Col 3
Row 3 Col 1	Row 3 Col 2	Row 3 Col 3

Here is after table.
